File: server/NasaSuits_051718/api/models/IU.php
<?php
class IU
{
    var $iu_id;
    var $iu_link;
    var $iu_type;
    var $iau_id;
	var $iu_position;
	var $iu_rotation;
	var $iu_scale;
	var $iu_space;
    

    function __construct($iu_id, $iu_link, $iu_type, $iau_id, $iu_position, $iu_rotation, $iu_scale, $iu_space)
    {
        $this->iu_id = $iu_id;
        $this->iu_link = $iu_link;
        $this->iu_type = $iu_type;
		$this->iau_id = $iau_id;
		$this->iu_position = $iu_position;
		$this->iu_rotation = $iu_rotation;
		$this->iu_scale = $iu_scale;
		$this->iu_space = $iu_space;
    }
}

File: unity3d/Sources/Assets/Scripts/EVAModel/IU.cs
using System.Collections;
using System.Collections.Generic;
using System.Globalization;
using UnityEngine;

public class IU
{

    public string iu_id, iu_link, iu_type, iau_id, iu_position, iu_rotation, iu_scale, iu_space;

    public Vector3 GetPosition()
    {
        string[] ordinates = iu_position.Split(',');
        float x = float.Parse(ordinates[0], CultureInfo.InvariantCulture.NumberFormat);
        float y = float.Parse(ordinates[1], CultureInfo.InvariantCulture.NumberFormat);
        float z = float.Parse(ordinates[2], CultureInfo.InvariantCulture.NumberFormat);
        return new Vector3(x,y,z);
    }

    public Quaternion GetRotation()
    {
        string[] rotations = iu_rotation.Split(',');
        float x = float.Parse(rotations[0], CultureInfo.InvariantCulture.NumberFormat);
        float y = float.Parse(rotations[1], CultureInfo.InvariantCulture.NumberFormat);
        float z = float.Parse(rotations[2], CultureInfo.InvariantCulture.NumberFormat);
        return Quaternion.Euler(x, y, z);
    }

    public Vector3 GetScale()
    {
        string[] scales = iu_scale.Split(',');
        float x = float.Parse(scales[0], CultureInfo.InvariantCulture.NumberFormat);
        float y = float.Parse(scales[1], CultureInfo.InvariantCulture.NumberFormat);
        float z = float.Parse(scales[2], CultureInfo.InvariantCulture.NumberFormat);
        return new Vector3(x, y, z);
    }

}
